hbase-common/src/main/java/org/apache/hadoop/hbase/KeyValueUtil.java

```diff
@@ -48,8 +46,6 @@
 @InterfaceAudience.Private
 public class KeyValueUtil {
 
-  private static final Logger LOG = LoggerFactory.getLogger(KeyValueUtil.class);
```

Comment:
Great.  Thanks.  This goes under the heading of 'log and throw'.

Reply:
that's well and good in principle but only works if we actually log something about the exception when it's eventually caught. also depending on the severity it might be needed if we're at risk of stacktrace optimization by the JVM and log rolling making it so we can't actually get details from here when it comes time to log the exception elsewhere.